src/app/components/forms/form-base.ts

```diff
@@ -6,7 +6,10 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 })
 export class FormBaseComponent {
   @Input() id!: string;
-  @Input() label: string = '';
+  @Input() label!: string;
+  @Input() maxLength!: string;
+  @Input() width!: string;
```

Comment:
Deze non-null assertions lijken me geen goed idee aangezien deze velden best leeg kunnen zijn. Mochten de velden required moeten zijn, daar heb ik deze thread over gevonden: https://github.com/angular/angular/issues/18156#issuecomment-315774348

Maar ik zie verderop een `*ngIf` op label. Dus dan kan dit `@Input() label?: string;` zijn, toch?

Reply:
Voor zover ik begrijp betekent het ! dat hij undefined is op initialization. Ik zie geen situatie waarbij ze `null` kunnen zijn aangezien wij ze zelf zetten in de html?

Reply:
We gaan ze nooit met `null` zetten. Het is: of niet meegeven of meegeven met een string waarde